shell/artifacts/Products/source/GiftList.js

```diff
@@ -33,15 +33,19 @@ defineParticle(({DomParticle}) => {
     get template() {
       return template;
     }
+    _shouldRender(props, state) {
+      return Boolean(props.person);
+    }
     _render(props, state) {
```

Comment:
Similar, can just be `_render({person})` and simplifies below and no refs to `state`.

Reply:
done